src/commands/dev.ts

```diff
@@ -418,8 +419,42 @@ export async function command(commandOptions: CommandOptions) {
     }
   });
 
-  const server = http
-    .createServer(async (req, res) => {
+
+  const readCredentials = async () => {
+    const [cert, key] = await Promise.all([fs.readFile(__dirname + '/snowpack.crt'),fs.readFile(__dirname + '/snowpack.key')])
+    
+    return {
+      cert,
+      key,
+    }
+  };
+  
+  const certify = () =>
+  execa.sync(__dirname + '/certify.sh', {
+      cwd: __dirname,
+    });
+
+    
+  let credentials
+  if (config.devOptions.secure) {
+    try {
+      credentials = await readCredentials();
+    } catch (e) {
+      certify();
+      try {
+        credentials = await readCredentials();
+      } catch (e) {
+        console.log(
+          `\n  ${chalk.yellow('⚠️  There was a problem generating ssl credentials. Try removing `--secure`\n')}`
```

Comment:
If this simple credential generation doesn't work, we should point the user to run https://github.com/FiloSottile/mkcert (or devcert) themselves, which seems setup to handle more complex workflow

Reply:
Do you think it would be useful to link to a spot in the snowpack docs that would have resources for troubleshooting?

I found this gist that is pretty informative and useful if you don't want to install a tool. https://gist.github.com/cecilemuller/9492b848eb8fe46d462abeb26656c4f8

Reply:
Maybe we even call it HTTPS/HTTP2 and highlight the perf benefit as well for this

Reply:
well... I attempted to write some docs. I'm pretty terrible at writing 😄  and I wasn't sure what to write in the HTTP2/perf section

Reply:
np! I'll take an edit pass after merging